src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = 40 * days;
```

Comment:
Consider using a constant for the daily rental cost instead of the literal `40` to make the value's purpose clear and easily changeable.